smartseq2_single_sample/pipelines/run_pipeline_scatter.wdl

```diff
@@ -0,0 +1,53 @@
+import "hisat2_QC_pipeline.wdl" as run_hisat2
+import "hisat2_rsem_pipeline.wdl" as run_hisat2_rsem
+
+workflow run_pipelines {
+
+  # load annotation
+  File gtf_file
+  File genome_ref_fasta
+  File rrna_intervals
+  File gene_ref_flat
+  #load index
+  File star_ref_index
+  File hisat2_ref_index
+  File hisat2_ref_trans_index
+  File rsem_ref_index
+  File kallisto_ref_index
+  # ref index name
+  File hisat2_ref_name
+  File hisat2_ref_trans_name
+  # array of string represents featuretypes
+  # samples
+  String stranded
+  File sample_info
+  String sra_dir
+  Array[String] sraIDs = read_lines(sample_info)
+  
+  scatter(idx in range(length(sraIDs))) { 
+    call run_hisat2.RunHisat2Pipeline {
+      input:
+        fastq_read1 = sra_dir+'/'+sraIDs[idx]+"_1.fastq.gz",
+        fastq_read2 = sra_dir+'/'+sraIDs[idx]+"_2.fastq.gz", 
```

Comment:
This seems pretty tightly coupled to a particular bucket. How would this work outside of the Broad infrastructure?

Reply:
I thiiiink this is a wdl just to run the test pipeline for analysis, so maybe not as concerning that it's portable. But fair point.